docs/collect_readmes.py

```diff
@@ -168,8 +169,29 @@ def handle_default(block):
         return handle_default_with_literals(block)
 
     ret = {}
+
+
+    final_text = ""
+    if block.tagname.strip() == "paragraph":
+
+        skip_next = False
+        for node in block.traverse():
+            if skip_next:
+                skip_next = False
```

Comment:
When this situation can happen? You are setting `skip_next` to `False` two lines above.

Reply:
Fixed. Removed this and rebased on top of master which has enhanced parsing of inline nodes